pkgs/servers/mastodon/yarn.nix

```diff
@@ -0,0 +1,34 @@
+{
+  stdenvNoCC,
+  yarn-berry,
+  cacert,
+  src,
+  hash,
+}:
+stdenvNoCC.mkDerivation {
+  name = "yarn-deps";
```

Comment:
```suggestion
  pname = "yarn-deps";
  inherit version;
```

Reply:
error:
```
       … while calling the 'derivationStrict' builtin
         at <nix/derivation-internal.nix>:34:12:
           33|
           34|   strict = derivationStrict drvAttrs;
             |            ^
           35|

       … while evaluating derivation 'mastodon-4.3.0'
         whose name attribute is located at /home/lafiel/works/src-nix/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:336:7

       … while evaluating attribute 'mastodonModules' of derivation 'mastodon-4.3.0'
         at /home/user/works/src-nix/nixpkgs/pkgs/servers/mastodon/default.nix:26:3:
           25|
           26|   mastodonModules = stdenv.mkDerivation {
             |   ^
           27|     pname = "${pname}-modules";

       (stack trace truncated; use '--show-trace' to show the full, detailed trace)

       error: undefined variable 'version'
       at /home/user/works/src-nix/nixpkgs/pkgs/servers/mastodon/yarn.nix:10:11:
            9|   pname = "yarn-deps";
           10|   inherit version;
             |           ^
           11|   nativeBuildInputs = [
```

Reply:
We also need to add it to the inputs here and inherit it.